pkg/api/api.go

```diff
@@ -7,6 +7,7 @@ const (
 
 // consts to maintain existing names of various sub-resources
 const (
+	ClusterOperatorName                 = "console"
```

Comment:
This will most probably conflict with https://github.com/openshift/console-operator/pull/262

Reply:
Looks like its not a prob.  No rebase tag.